linux_os/guide/services/ssh/ssh_server/sshd_disable_root_login/rule.yml

```diff
@@ -26,6 +26,7 @@ identifiers:
 references:
     anssi@debian8: NT007(R21)
     anssi@debian9: NT007(R21)
+    anssi@debian10: NT007(R21)
```

Comment:
Since all the anssi mappings are the same, please collapse into a single entry e.g. `anssi: NT007(R21)`